packages/array-standard-deviation/src/__tests__/test.js

```diff
@@ -0,0 +1,8 @@
+import standardDeviation from '..';
+
+var data = [15, 13, 17, 7];
+describe('standard deviation', () => {
+    var s = standardDeviation(data);
+    expect(standardDeviation(data, true)).toBe(s);
```

Comment:
IMO we should make two separate checks. One without the option and one with it and use the real value to check if the result is correct